presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -326,12 +325,11 @@ public void testIsNotNullFilter()
     {
         assertExpression("x IS NOT NULL")
                 .outputRowsCount(750.0)
-                .symbolStats("x", symbolStats -> {
-                    symbolStats.distinctValuesCount(40.0)
-                            .lowValue(-10.0)
-                            .highValue(10.0)
-                            .nullsFraction(0.0);
-                });
```

Comment:
i liked the original form more. I don't object though.

Reply:
It wasn't consistent within the file and IntelliJ hints this way